data/data/vsphere/pre-bootstrap/outputs.tf

```diff
@@ -11,7 +11,7 @@ output "folder" {
 }
 
 output "network" {
```

Comment:
This output should be removed. Later stages that were using this output should be changed to use `vsphere_network` instead.

Also, the description of the `vsphere_network` variable should be updated to reflect that it is the ID now rather than the name.

Reply:
Thanks, both points addressed in f10b387455edeb5314646abfe5f8231b8fa34e74